README.md

```diff
@@ -1,4 +1,4 @@
-Zen 3.3.1
+Zen 4.0.99
```

Comment:
I wonder if this version shouldn't be `4.1.99` considering that we are currently working on features planned for `v4.1.0`.

What do you think?

Reply:
I didn't consider this perspective at the beginning but it definitely makes sense: it feels more "natural" switching from `4.0.99` to `4.1.0` when creating the next release compared to switching from `4.1.99` back to `4.1.0`.

It's still possible, however, that we will end up having to release `4.0.1` or `4.0.2`, but typically the release preparation should start from `v4.0.x` branch with some eventual cherry-picks.

All in all, I like the proposed approach, let's keep it as it is, thanks!